src/content/docs/synthetics/synthetic-monitoring/getting-started/types-synthetic-monitors.mdx

```diff
@@ -70,6 +70,26 @@ These are the four types of synthetic monitors:
       </td>
     </tr>
 
+    <tr>
+      <td>
+        Step monitor
+
+        API name: `STEP_MONITOR`
+      </td>
+
+      <td>
+        Step Monitors are advanced monitors which require no code to set up. 
```

Comment:
Recommend double-check consistency of "Step Monitor" or "Step monitor" or "step monitor" throughout

Reply:
Just pushed up a commit fixing this, thanks!